### PR TITLE
Creating the routes of the project

### PR DESCRIPTION
**Pull request by jlandwehr-avarteq**
This project was just created

This is the initialization of a Hello_Wolrd project.
I edited the index.thml.erb-view and added
a root to the welcome_controller.

### CODE REVIEW

**Comment by arkanoryn** [comment]
@jlandwehr-avarteq you have a typo on your commit. Please fix it.
On line 5 you have a space before your word...

File: blog/app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
+  def index
+  end
+
```
Comment (arkanoryn) [inline blog/app/controllers/welcome_controller.rb]: why an empty line?

File: blog/app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Blog</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+
```
Comment (arkanoryn) [inline blog/app/views/layouts/application.html.erb]: empty line?

File: blog/app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<%= yield %>
+
```
Comment (arkanoryn) [inline blog/app/views/layouts/application.html.erb]: empty line?

File: blog/app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+<html>
+<head>
+  <title>Blog</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+
+<%= yield %>
```
Comment (arkanoryn) [inline blog/app/views/layouts/application.html.erb]: indentation?

File: blog/app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (arkanoryn) [inline blog/app/views/layouts/application.html.erb]: indentation?

File: blog/config/secrets.yml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
```
Comment (arkanoryn) [inline blog/config/secrets.yml]: This file should be a `.secret.yml.example` and the `.secret.yml` should be removed from git and put in the Git `.ignore` file.

Remember it for the next projects.

**Comment by arkanoryn** [comment]
@jlandwehr-avarteq I commented over for some changes.
Please fix them before merging into master.

Also, rebase your current branch on top of master as soon as master has been updated.

**Review by arkanoryn — APPROVED** [review]
Ok for me.
Please, remove the `.DS_Store` files before you merge on master.

You can then merge it on master.
Once merge on master, do not forget to `rebase` all other existing branch on top of master.
If you want, we can do it one time together this afternoon.

**Comment by jlandwehr-avarteq** [comment]
Merge done.

**Pull request closed without merging**